mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -41,65 +43,183 @@ description: Vertex AI Feature Online Store provides a centralized repository fo
 autogen_async: false
 examples:
   - !ruby/object:Provider::Terraform::Examples
-    name: vertex_ai_feature_online_store
-    primary_resource_id: feature_online_store
+    name: 'vertex_ai_featureonlinestore_with_bigtable'
+    primary_resource_id: 'featureonlinestore'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
```

Comment:
I don't think you need this field unless there are IAM resources being generated. Same for other examples

Reply:
Thanks! I see "primary_resource_name" is not being used in my examples. Updated!